src/codecs/avif/yuv.rs

```diff
@@ -610,6 +703,125 @@ fn process_halved_chroma_row<
     }
 }
 
+fn process_halved_chroma_row_cgco<
+    V: Copy + AsPrimitive<i32> + 'static + Sized,
+    const PRECISION: i32,
+    const CN: usize,
```

Comment:
In many other instances this is spelled out:

```suggestion
    const CHANNELS: usize,
```